src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+            this._interval &&
+              (clearInterval(this._interval), (this._interval = null)),
+            this._config.interval &&
+              !this._isPaused &&
+              (this._interval = setInterval(
+                (document.visibilityState
+                  ? this.nextWhenVisible
+                  : this.next
+                ).bind(this),
+                this._config.interval
+              ));
+        }),
+        (n.to = function (t) {
+          var n = this;
+          this._activeElement = this._element.querySelector(
+            ".active.carousel-item"
+          );
+          var i = this._getItemIndex(this._activeElement);
+          if (!(t > this._items.length - 1 || t < 0))
+            if (this._isSliding)
+              e(this._element).one("slid.bs.carousel", function () {
+                return n.to(t);
+              });
+            else {
+              if (i === t) return this.pause(), void this.cycle();
+              var o = t > i ? "next" : "prev";
+              this._slide(o, this._items[t]);
+            }
+        }),
+        (n.dispose = function () {
+          e(this._element).off(m),
+            e.removeData(this._element, "bs.carousel"),
+            (this._items = null),
+            (this._config = null),
+            (this._element = null),
+            (this._interval = null),
+            (this._isPaused = null),
+            (this._isSliding = null),
+            (this._activeElement = null),
+            (this._indicatorsElement = null);
+        }),
+        (n._getConfig = function (t) {
+          return (t = s(s({}, v), t)), l.typeCheckConfig(p, t, _), t;
+        }),
+        (n._handleSwipe = function () {
+          var t = Math.abs(this.touchDeltaX);
+          if (!(t <= 40)) {
+            var e = t / this.touchDeltaX;
+            (this.touchDeltaX = 0), e > 0 && this.prev(), e < 0 && this.next();
+          }
+        }),
+        (n._addEventListeners = function () {
+          var t = this;
+          this._config.keyboard &&
+            e(this._element).on("keydown.bs.carousel", function (e) {
+              return t._keydown(e);
+            }),
+            "hover" === this._config.pause &&
+              e(this._element)
+                .on("mouseenter.bs.carousel", function (e) {
+                  return t.pause(e);
+                })
+                .on("mouseleave.bs.carousel", function (e) {
+                  return t.cycle(e);
+                }),
+            this._config.touch && this._addTouchEventListeners();
+        }),
+        (n._addTouchEventListeners = function () {
+          var t = this;
+          if (this._touchSupported) {
+            var n = function (e) {
+                t._pointerEvent && b[e.originalEvent.pointerType.toUpperCase()]
+                  ? (t.touchStartX = e.originalEvent.clientX)
+                  : t._pointerEvent ||
+                    (t.touchStartX = e.originalEvent.touches[0].clientX);
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)